src/app/backend/deploy.go

```diff
@@ -86,13 +96,23 @@ func DeployApp(spec *AppDeploymentSpec, client *client.Client) error {
 		Labels:      labels,
 	}
 
+	containerSpec := api.Container{
+		Name:  spec.Name,
+		Image: spec.ContainerImage,
+	}
+
+	if spec.ContainerCommand != nil {
+		containerSpec.Command = []string{*spec.ContainerCommand}
+	}
+
+	if spec.ContainerCommandArgs != nil {
+		containerSpec.Args = []string{*spec.ContainerCommandArgs}
```

Comment:
As backend actually has array of args and array of commands shouldn't we support it? Now there is array with 1 element only.